presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -386,6 +386,12 @@ public void finishInsertTable(JdbcOutputTableHandle handle)
         }
     }
 
+    @Override
+    public JdbcPageSink getPageSink(JdbcOutputTableHandle tableHandle)
+    {
+        return new JdbcPageSink(tableHandle, this);
```

Comment:
Instead, I would rather extend `ReadMapping` into `ColumMapping`, adding additional information. This way, a connector extending base-jdbc will need to declare its requirements instead of the actual implementation, reducing coupling.